sources/packages/api/src/route-controllers/institution-locations/institution-locations.controller.ts

```diff
@@ -230,10 +231,26 @@ export class InstitutionLocationsController extends BaseController {
       );
     }
     // get all institution locations.
-    return this.locationService.getInstitutionLocation(
-      institutionDetails.id,
-      locationId,
-    );
+    const institutionLocation: InstitutionLocation =
```

Comment:
The given service method was returning DB entity modal. refactored it to return DTO.